dapps/marketplace/src/components/growth/ActionList.js

```diff
@@ -140,18 +135,18 @@ class ActionList extends Component {
               'completed',
               isMobile
             )}
-          </div>
+          </div>*/}
           <div className="d-flex flex-column">
             {title !== undefined && <div className="action-title">{title}</div>}
-            {actionsToDisplay.map(action => {
+            {actionsToDisplay.map((action, i) => {
               return (
                 <Action
                   action={action}
+                  hasBorder={i !== actionsToDisplay.length - 1}
```

Comment:
I think we should try to avoid style related props. Instead, you can target with css by using pseudo selectors such as [`:last-of-type`](https://www.w3schools.com/cssref/sel_last-of-type.asp)... eg to add border and padding to all but the last element with a particular class you can do:

```
.styled-row
  &:not(:last-of-type)
    padding-bottom: 1rem
    border-bottom: 1px solid red

```

Reply:
Good point, much cleaner tnx